src/App.js

```diff
@@ -14,11 +15,32 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    started: false,
+  }
+
+  render() {
+    return (
+      <div className="App">
+        {!this.state.started
```

Comment:
```suggestion
  render() {
    const { started } = this.state;

    return (
      <div className="App">
        {!started
```
ось так має працювати